src/mem/localalloc.h

```diff
@@ -426,7 +426,7 @@ namespace snmalloc
       // make pass through call aligned_alloc with the alignment snmalloc
       // would guarantee.
       void* result = external_alloc::aligned_alloc(
-        natural_alignment(size | (sizeof(void*) - 1)), round_size(size));
+        natural_alignment(size), round_size(size));
```

Comment:
Did you mean to revert this hunk from 966fe9df74f4b337b11c0ce0e7e08cacfe1716ce?

Reply:
Yeah, this was actually incorrect.  I don't know how it passed locally.